lnrpc/autopilotrpc/autopilot_server.go

```diff
@@ -63,7 +65,7 @@ type ServerShell struct {
 // RPC server allows external callers to access the status of the autopilot
 // currently active within lnd, as well as configuring it at runtime.
 type Server struct {
-	started  int32 // To be used atomically.
+	injected int32 // To be used atomically.
```

Comment:
You can use [atomic.Int32](https://pkg.go.dev/sync/atomic#Int32) to get compile-time guarantees that it is used atomically.